src/ukify/ukify.py

```diff
@@ -370,6 +396,19 @@ def combine_signatures(pcrsigs):
     return json.dumps(combined)
 
 
+def key_path_groups(opts):
+    if not opts.pcr_private_keys:
+        return
+
+    n_priv = len(opts.pcr_private_keys or ())
```

Comment:
```suggestion
    n_priv = len(opts.pcr_private_keys)
```
since if `opts.pcr_private_keys` is falsey, it will return before this line